esp/src/src-react/components/WorkunitSummary.tsx

```diff
@@ -160,12 +162,27 @@ export const WorkunitSummary: React.FunctionComponent<WorkunitSummaryProps> = ({
             key: "slaveLogs", text: nlsHPCC.SlaveLogs, disabled: !workunit?.ThorLogList,
             onClick: () => setShowThorSlaveLogs(true)
         },
-    ], [_protected, canDelete, canDeschedule, canReschedule, canSave, description, jobname, refresh, setShowDeleteConfirm, workunit, wuid]);
+    ], [_protected, canDelete, canDeschedule, canReschedule, canSave, description, jobname, refresh, refreshSavings, setShowDeleteConfirm, workunit, wuid]);
 
     const serviceNames = React.useMemo(() => {
         return workunit?.ServiceNames?.Item?.join("\n") || "";
     }, [workunit?.ServiceNames?.Item]);
 
+    const totalCosts = React.useMemo(() => {
+        return (workunit?.CompileCost ?? 0) +
+            (workunit?.ExecuteCost ?? 0) +
+            (workunit?.FileAccessCost ?? 0);
+    }, [workunit?.CompileCost, workunit?.ExecuteCost, workunit?.FileAccessCost]);
+
+    const potentialSavings = React.useMemo(() => {
+        return exceptions.reduce((prev, cur) => {
+            if (!isNaN(cur.Priority)) {
```

Comment:
Might should be something like `!isNaN(parseInt(cur.Priority, 10))`, because there are weird positive results for !isNaN(x) where x is null or "" and such